content/04.results.md

```diff
@@ -6,7 +6,9 @@ Results section stub.
 
 This section will introduce the dataset (e.g., the histologies represented and what data types are included; Figure {@fig:openpbta-overview}A-B) and the process for contributing analytical code and to the manuscript (Figure {@fig:openpbta-overview}C-D).
 
-![An overview of the OpenPBTA project](images/figure_sketches/fig1-proposed-sketch.png){#fig:openpbta-overview}
+![An overview of the OpenPBTA project sketch](images/figure_sketches/fig1-proposed-sketch.png){#fig:openpbta-overview-sketch}
```

Comment:
Can you remove this please? I would also remove the PNG of the sketch from the repository.